### DISCUSSION TITLE
PnP Search Results V4 Path Query Template

### DISCUSSION THREAD

**Discussion by jckicks2lt** [microsoft-search/pnp-modern-search#2943] [General]
Hi All,

I noticed when just using {search terms} in the query template and local sharepoint results renders a blank canvas before a search is performed in the search box. Then upon entering search terms you receive your results.  When I try to add a path and combine it with the {search terms} expression i noticed the path displays prior to search in the search results webpart.  Is there a way I can hide the library and results the same way the search local sharpeoint results hides results prior to submitting and what would be the best way to show this?

**Comment by wobba** [comment]
See https://github.com/microsoft-search/pnp-modern-search/issues/1721#issuecomment-1502226663
